<launch>
 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="iroad_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="accel_id" value="5" type="int"/>
  <param name="steer_id" value="0" type="int"/>
  <param name="ct_input_id" value="2" type="int"/>
  <param name="PRNDL_vr_id" value="1" type="int"/>
  <param name="PRNDL_ct_id" value="2" type="int"/>
  <param name="pk_brake_id" value="0" type="int"/>
  <param name="steer_scale" value="350" type="double"/>
  <param name="origin_lat" value="37.4303024" type="double"/>
  <param name="origin_lon" value="-122.1839708" type="double"/>
  <param name="origin_alt" value="0.0" type="double"/>
  <node pkg="bridge_iroad" type="iroad_teleop_joy" name="iroad_teleop_joy" output="screen"/>
</launch>
